packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -292,7 +292,7 @@ const getStyles = (theme: GrafanaTheme2) => {
     content: css({
       label: 'panel-content',
       flexGrow: 1,
-      contain: 'strict',
+      // contain: 'strict',
```

Comment:
this is needed

Reply:
the tooltip is in a portal, so why did you need to change this?

Reply:
this was only needed for text/html/markdown panel to prevent malicious contents from exiting the panel. it was done globally in kay's original PR but not really by design, afaik.

we will still have this for panels that can have user-supplied content.

Reply:
> the tooltip is in a portal, so why did you need to change this?

the tooltip is portaled into uplot's dom (not the global portal), so we need it to exit the panel bounds

Reply:
e.g.: https://github.com/grafana/support-escalations/issues/6509

also:

https://github.com/grafana/grafana/assets/43234/11f21bc1-99b0-4318-a5b7-e06870df65bf

Reply:
@leeoniya @adela-almasan you will need to test a lot of external plugins and make sure they don't overflow the panel, as that is what ``contain: 'strict',``  fixed.  

But maybe there is another way to solve it, we have had contain or overflow: auto for many many years

Reply:
but if contain: strict was only added to prevent " which could previously be used in conjunction with the text panel and position: fixed;, for example, to arbitrarily place content on the screen."   then we could fix that in TextPanel instead of globally here

Reply:
@torkelo that's what me and @leeoniya were discussing, since the changes were initially needed just for `TextPanel` but ended up being applied globally - lower the scope of `contain: strict`, use it maybe in a wrapper (+sanitization?) and apply it selectively to the panels that are expecting user content. We are not fully aware of all the implications yet, but I will look into this this issue next week.

Reply:
great, yea I think it should be fine. There was a period (~2 years) when we did not have overflow: auto or contain: 'strict'

Reply:
these changes are now in `main`